cristin-project/src/test/java/no/unit/nva/cristin/projects/model/nva/NvaProjectCristinQueryBuilderTest.java

```diff
@@ -16,7 +16,7 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class NvaProjectBuilderTest {
+public class NvaProjectCristinQueryBuilderTest {
```

Comment:
Why has this been renamed? NvaProjectBuilder is a model transform and has nothing to do with query params

Reply:
This is a auto renaming error, me fix